source/beamline/CLS/CLSBeamlineStatus.cpp

```diff
@@ -10,18 +13,20 @@ CLSBeamlineStatus::CLSBeamlineStatus(const QString &name, QObject *parent) :
 	monoMaskControlSet_ = new AMControlSet(this);
 
 	// Setup the basic value options.
-	addOption(On, "On", true);
-	addOption(Off, "Off", true);
+	addOption(On, "Beam is ON", true);
+	addOption(Off, "Beam is OFF", true);
+
+	connect(this, SIGNAL(connected(bool)), this, SLOT(updateBeamlineStatus()));
```

Comment:
Given the option, I would prefer to reimplement the `AMEnumeratedControl::updateStates()` or `updateValue()` or something, and add whatever additional functionality is needed there (emitting the new `beamlineStatusChanged(bool)` signal).